internal/operators/manager_test.go

```diff
@@ -161,6 +178,36 @@ var _ = Describe("Operators manager", func() {
 		)
 	})
 
+	Context("EnsureLVMAndCNVNotEnabled", func() {
```

Comment:
What do these unit tests check?  They should be calling RegisterCluster and UpdateCluster, and making sure that the proper errors are returned.